README.md

```diff
@@ -0,0 +1,51 @@
+# Aragon Court Dashboard <a href="https://aragon.org/"><img align="right" src="docs/assets/LogoAccent.svg" height="80px" /></a>
+
+### Aragon Court handles subjective disputes that require the judgment of human jurors.
+
+<a href="https://court.aragon.org/dashboard"><img src="docs/assets/screenshot.png" /></a>
```

Comment:
Ah... I think Github might have a problem with case sensitive files.

It looks like the new file has an upper case `S`. Fixing this might be a bit of a pain (by default macOS is file-case insensitive, although APFS might have changed that), but worst case we can apply some rebasing and it will likely work.

Reply:
Oops, used [`git mv`](https://git-scm.com/docs/git-mv) and it recognized the modification.